app/controllers/SwitchLanguageController.scala

```diff
@@ -23,11 +23,27 @@ import config.AppConfig
 import uk.gov.hmrc.play.language.{LanguageController, LanguageUtils}
 
 @Singleton
-class SwitchLanguageController @Inject() (appConfig: AppConfig, languageUtils: LanguageUtils, cc: MessagesControllerComponents)
+class SwitchLanguageController @Inject(
+) (appConfig: AppConfig, languageUtils: LanguageUtils, cc: MessagesControllerComponents)
```

Comment:
I'd move the closing parenthesis up to @Inject(

Reply:
:)